ospd_openvas/daemon.py

```diff
@@ -235,15 +235,12 @@ class OpenVasVtsFilter(VtsFilter):
     """
 
     def format_vt_modification_time(self, value):
-        """ Convert the datetime value in an 19 character string
-        representing YearMonthDateHourMinuteSecond.
+        """ Convert the string seconds since epoch into a 19 character
+        string representing YearMonthDayHourMinuteSecond.
```

Comment:
It doesn't state anything about the timezone of the returned string. We should make it clear in which timezone this string representation is in. I would prefer UTC.